### PR TITLE
#919 Add a Markdown tutorial in githubandmarkdown.md

### PR DESCRIPTION
**Pull request by yoeng327**
This PR resolves issue #919
Added a Markdown tutorial in githubandmarkdown.md.
[rawgit](https://rawgit.com/yoeng327/yoeng327.github.io/tutorial-branch-yoeng327/#!pages/githubandmarkdown.md)

screen shot 2017-06-24 at 11 16 01 pm

### CODE REVIEW

File: pages/firststeps.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ Review these [Vagrant instructions](vagrant.md) to ensure that you have fully co
 
 Follow the instructions on the [GitHub and Markdown page](githubandmarkdown.md). Make sure that you've linked to your github.io and your personal .md page on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (&lt;username&gt;.github.io and &lt;username&gt;.github.io/#!pages/profiles/&lt;username&gt;.md).
 
+Here is the tutorial as reference to work on the Step 3 and Step 5.
+Markdown Tutorial:
```
Comment (kylemathias) [inline pages/firststeps.md]: Delete this line. just use the link below.

File: pages/firststeps.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ Review these [Vagrant instructions](vagrant.md) to ensure that you have fully co
...
+[MarkdownTutorial](https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet#links)
```
Comment (kylemathias) [inline pages/firststeps.md]: add a ":" after MarkdownTutorial, so it looks like this "Markdown Tutorial:"

File: pages/firststeps.md (hunk 1 above)
Comment (yoeng327) [inline pages/firststeps.md]: I edited what you told me to do. Could you really approve the pull request please?

File: pages/firststeps.md (hunk 2 above)
Comment (yoeng327) [inline pages/firststeps.md]: Hello. I edited what you told me to do. Could you really approve the pull request please?

**Comment by sarah-lu102** [comment]
capture I would put the link where the other markdown tutorial links are

File: pages/firststeps.md [hunk 3]
```diff
@@ -45,6 +45,8 @@ Review these [Vagrant instructions](vagrant.md) to ensure that you have fully co
 
 Follow the instructions on the [GitHub and Markdown page](githubandmarkdown.md). Make sure that you've linked to your github.io and your personal .md page on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (&lt;username&gt;.github.io and &lt;username&gt;.github.io/#!pages/profiles/&lt;username&gt;.md).
 
+[Markdown Tutorial](https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet#links)
```
Comment (sarah-lu102) [inline pages/firststeps.md]: can you delete this line?

Comment (yoeng327) [inline pages/firststeps.md]: Hello.
I got removed the line in firststeps.md.
I added the tutorial's link in "githubandmarkdown.md"
Could you take a look please?
Could you please approve this pull request please?

**Comment by yoeng327** [comment]
Hello.
screen shot 2017-06-26 at 5 00 19 pm
screen shot 2017-06-26 at 5 09 02 pm

I got removed the line in firststeps.md.
I added the tutorial's link in "githubandmarkdown.md"
Could you take a look please?
Could you please approve this pull request please?

**Pull request merged**